### ISSUE TITLE
Allegro foreign marketplaces - delivery price list

### ISSUE THREAD

**Issue by dominikahr** [allegro/allegro-api#9473]
Hi, 

I was wondering if it is somehow possible to check, if the seller has included additional storefront in his shipping price list. So that we would know from the beggining, even before he will publish the offers on his base storefront, that he wants to sell on another market as well. 

Because when he already has two countries in his shipping price list before he published any offers, then the offers once active will be visible on both markets from the beggining, right? 

Thanks in advance! 

All the best, 

Dominika

**Comment by AureliuszBrussy** [comment]
Yes, that's right, if there are proper delivery methods in the price list, it will be available on additional marketplaces from the very beginning. 

Before the listing itself, it is possible to identify and check the delivery price list, which will be used by the seller when creating the product offer. You will do this using the endpoint [GET /sale/shipping-rates](https://developer.allegro.pl/documentation#operation/getListOfShippingRatestUsingGET).
